### DISCUSSION TITLE
php artisan native:install

### DISCUSSION THREAD

**Discussion by malikbrwa** [NativePHP/laravel#193] [Q&A] [answered]
When the " composer require nativephp/electron" code.
run below version nativephp does not show when the code " php artisan native:install"
I run it in blue (Publishing NativePHP Service Provider...) and it will stay on for a long time

![Screenshot 2023-08-25 212135](https://github.com/NativePHP/laravel/assets/143215319/43fe5601-d617-47e1-9c67-1fcfb529fbda)

**Comment by eskayamadeus** [comment]
Give it some time to load. I'm sure future releases will be faster.

**Comment by simonhamp** [comment, marked as answer]
This was a bug that was recently fixed. Please upgrade to the latest version of `nativephp/electron`.
